app-practice/ViewController.swift

```diff
@@ -8,18 +8,51 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, UITextFieldDelegate {
+    
+    //MARK: Properties
+    var txt = "Enter text!"
+    @IBOutlet weak var greetingLabel: UILabel!
+    @IBOutlet weak var greetingTextField: UITextField!
+    
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        
+        // Handle the text field's user input through delegate callbacks.
+        greetingTextField.delegate = self
```

Comment:
Why did you put this line?

Reply:
I think this is so that if you use the keyboard on the phone, it can handle the text entered through this way and hitting enter rather than pressing the button itself (so there are two ways to enter the text)